app/components/provider_interface/filter_component.html.erb

```diff
@@ -5,6 +5,7 @@
         <h2 class="govuk-heading-m">Filter</h2>
       </div>
       <div class="moj-filter__header-action">
+        <%= link_to("Close", provider_interface_applications_path(toggle_filter_off_params),class:"moj-filter__close", type:"button") %>
```

Comment:
I think this is by design - on larger screens the filter should always be present and there should be no way to hide it. On smaller screens you are able to show and hide the filter.

@adamsilver this is how I understood the UI flow - is this what you intended?

Reply:
The point you make about people re-orientating devices and not being able to show / hide filters is a really interesting one. I'll let @adamsilver comment on it from a design perspective, but from a code point of view this would be easy to implement and as you say would allow for some UI testing of these feature. Thanks for highlighting this @mnacos 👍